pyproject.toml

```diff
@@ -24,6 +24,11 @@ dependencies = [
     "pyyaml",
     "click",
     "dill",
+    # TODO: move to [vis]? not required for cli-only usage?!
```

Comment:
Should the CLI also support plotting eventually?

Reply:
I'm currently not sure. I currently expect that after users generate their model selection results with e.g. pyPESTO, they then come to PEtab Select for any post-calibration analysis including visualization. This is not an issue for pyPESTO, since it's all within Python.

If tool developers want to provide these plots, I could add it to CLI. It might be easier for them to implement it themselves, though, using the plotting objects in the other languages, so for now I won't add CLI support.

Reply:
:+1: Then I'd move the visualization dependencies to `project.optional-dependencies` to simplify installation for those who use petab-select in some non-python context.